test/csuite/random_directio/main.c

```diff
@@ -153,7 +153,8 @@ static const char *const uri_rev = "table:rev";
  * that has schema operations happens again at id 200, assuming frequency
  * set to 100. So it is a good test of schema operations 'in flight'.
  */
-#define SCHEMA_OP_FREQUENCY 100
+/*#define SCHEMA_OP_FREQUENCY 100*/
+#define SCHEMA_OP_FREQUENCY 20
```

Comment:
If this is a temporary change, I'd put a comment here explaining.  Otherwise, probably shouldn't have commented out code.

Reply:
I changed this to take the frequency as a command line arg.